autometa/validation/datasets.py

```diff
@@ -0,0 +1,221 @@
+        if not os.path.exists(dir_path_size):
+            os.mkdir(dir_path_size)
+
+        for file_name in file_names:
+            file_id = df.query(f'file == "{file_name}"')["file_id"].to_list()[0]
+            dir_path_final = os.path.join(dir_path_size, file_name)
+            url = f"https://drive.google.com/uc?id={file_id}"
+
+            gdown.download(url, dir_path_final)
+
+
+def main():
+    import argparse
+    import logging as logger
+
+    logger.basicConfig(
+        format="[%(asctime)s %(levelname)s] %(name)s: %(message)s",
+        datefmt="%m/%d/%Y %I:%M:%S %p",
+        level=logger.DEBUG,
+    )
+
+    parser = argparse.ArgumentParser(
+        description="Download a simulated community file from google drive to a specified dir_path_size"
+    )
+    parser.add_argument(
+        "--community_type",
+        help="specify synthetic or simulated communities (currently only simulated is available)",
+        choices=[
+            "synthetic",
+            "simulated",
+            "all",
+        ],
+        required=True,
+    )
+    parser.add_argument(
+        "--community_size",
+        help="specify a community size to download from",
+        choices=[
+            "78Mbp",
+            "156Mbp",
+            "312Mbp",
+            "625Mbp",
+            "1250Mbp",
+            "2500Mbp",
+            "5000Mbp",
+            "10000Mbp",
+            "all",
+        ],
+        required=True,
+        nargs="+",
+    )
+    parser.add_argument(
+        "--file_names",
+        help="specify a file name to download",
+        choices=[
+            "README.md",
+            "reference_assignments.tsv.gz",
+            "metagenome.fna.gz",
+            "master.tsv.gz",
+            "control_reads.tsv.gz",
+            "control_contigs.tsv.gz",
+            "unclustered_recruitment.tsv.gz",
+            "binning.tsv.gz",
+            "taxonomy.tsv.gz",
+            "lengths.tsv.gz",
+            "coverages.tsv.gz",
+            "gc_content.tsv.gz",
+            "kmers.embedded.tsv.gz",
+            "kmers.tsv.gz",
+            "markers.tsv.gz",
+            "Bacteria.fna.gz",
+            "orfs.faa.gz",
+            "metagenome.filtered.fna.gz",
+            "hmmscan.tsv.gz",
+            "forward_reads.fastq.gz",
+            "reverse_reads.fastq.gz",
+            "all",
+        ],
+        nargs="+",
+        required=True,
+    )
+    parser.add_argument(
+        "--dir_path",
+        help="specify a folder to start the download (several directories will be generated within this folder)",
+        required=True,
+    )
+    args = parser.parse_args()
+
+    community_type = args.community_type
+    if "all" in args.community_size:
+        community_size = (
+            "78Mbp",
+            "156Mbp",
+            "312Mbp",
+            "625Mbp",
+            "1250Mbp",
+            "2500Mbp",
+            "5000Mbp",
+            "10000Mbp",
+        )
+    else:
+        community_size = args.community_size
```

Comment:
This should work for now, but as we add datasets, it may be more appropriate to retrieve all communities from the gdown file_ids table instead of hardcoding them in here. 

```suggestion
    if "all" in args.community_size:
        community_sizes = (
            "78Mbp",
            "156Mbp",
            "312Mbp",
            "625Mbp",
            "1250Mbp",
            "2500Mbp",
            "5000Mbp",
            "10000Mbp",
        )
    else:
        community_sizes = args.community_size
```

Reply:
I agree that I should revisit this in the future, but I know it would take me a long time to figure out how to do it dynamically, so it's probably a problem for later. There will be more added complexity when I add in the synthetic datasets, anyway.